src/models/user.js

```diff
@@ -22,10 +19,19 @@ export default (sequelize, DataTypes) => {
       primaryKey: true,
       autoIncrement: true,
     },
-    hsesUserId: DataTypes.STRING,
+    hsesUserId: {
+      type: DataTypes.STRING,
+      unique: true,
+    },
     name: DataTypes.STRING,
     phoneNumber: DataTypes.STRING,
-    email: DataTypes.STRING,
+    email: {
```

Comment:
Should this be unique? What happens if we have more than one user with the same email?